mne/stats/tests/test_regression.py

```diff
@@ -91,6 +91,7 @@ def test_continuous_regression_no_overlap():
     tmin, tmax = -.1, .5
 
     raw = mne.io.Raw(raw_fname, preload=True)
+    raw.apply_proj()
```

Comment:
We compare it to the regular evoked object and that one seemingly gets a projection applied (?). So this makes the objects we compare more similar. 